sdk/js/src/utils/repairVaa.ts

```diff
@@ -0,0 +1,137 @@
+import { ethers } from "ethers";
+import { CONTRACTS } from "./consts";
+import { Implementation__factory } from "../ethers-contracts";
+import { parseVaa, GuardianSignature } from "../vaa";
+import { hexToUint8Array } from "./array";
+import { keccak256 } from "../utils";
+
+const ETHEREUM_CORE_BRIDGE = CONTRACTS["MAINNET"].ethereum.core;
+
+function hex(x: string): string {
+  return ethers.utils.hexlify(x, { allowMissingPrefix: true });
+}
+
+export async function getGuardianSets(
+  provider: ethers.providers.JsonRpcProvider
+) {
+  let result: any = {};
```

Comment:
can we be a little more precise about the return type here? looks like it's actually a well-specified object